src/components/resources/CreateResource.js

```diff
@@ -113,6 +113,10 @@ const CreateResource = () => {
             setResourcesLinkError(true);
             setResourcesLinkErrorText('Please enter the link to your resource');
             return false;
+        } else if (!resourcesLink.includes('https://')) {
```

Comment:
this fix doesn't cover all cases, as a user may input "test https://" and it will lead to the previous issue. 

Here is a solution online that covers a similar issue:
https://stackoverflow.com/questions/5717093/check-if-a-javascript-string-is-a-url

Reply:
but why would they do that. URGH

Reply:
that was acc really useful thanks